src/main/java/net/mcreator/ui/component/JStringListField.java

```diff
@@ -45,6 +48,8 @@ public class JStringListField extends JPanel {
 	private final TechnicalButton edit = new TechnicalButton(UIRES.get("18px.edit"));
 	private final TechnicalButton clear = new TechnicalButton(UIRES.get("18px.removeall"));
 
+	private final List<ChangeListener> changeListeners = new ArrayList<>();

```

Comment:
Clear action should also fire change listener

Reply:
✔️ Fixed